src/main.c

```diff
@@ -2439,6 +2444,7 @@ void handle_movement(double dt) {
         }
     }
     float speed = g->flying ? 20 : 5;
+    if(isRunning) speed*=4;
```

Comment:
I thought a multiplier of 1.5 felt better when I tested it. Reminded me of minecrafts sprint; *4 seems like with speed boosts. But of course that is just preference